script/Deploy.s.sol

```diff
@@ -94,9 +94,31 @@ contract DeployMainnet is MainnetParams, Deploy {
 
     collateralTypes.push(WETH);
     collateralTypes.push(WSTETH);
+
+    // BUG: [FAIL. Reason: PIDRateSetter_InvalidPriceFeed()]
+
+    // _deployUniV3Pool();
+
+    // systemCoinOracle = uniV3RelayerFactory.deployUniV3Relayer({
+    //   _baseToken: address(systemCoin),
+    //   _quoteToken: address(collateral[WETH]),
+    //   _feeTier: HAI_POOL_FEE_TIER,
+    //   _quotePeriod: 1 days
+    // });
```

Comment:
ohhhh shit, does it try to read it on deployment? 😢

Reply:
That's it.

Reply:
we're merging w/o this comment no?

Reply:
Fixed.